bazel/ci/BUILD.bazel

```diff
@@ -3,10 +3,16 @@ load("@com_github_ash2k_bazel_tools//multirun:def.bzl", "multirun")
 load("@com_github_bazelbuild_buildtools//buildifier:def.bzl", "buildifier", "buildifier_test")
 load("//bazel/sh:def.bzl", "noop_warn", "repo_command", "sh_template")
 
-gazelle(name = "gazelle_generate")
+required_tags = ["e2e"]
+
+gazelle(
+    name = "gazelle_generate",
+    build_tags = required_tags,
```

Comment:
If `build_tags` is not set, gazelle does not generate build files for files that require a build tag. In this case, [e2e/internal/upgrade/BUILD.bazel](https://github.com/edgelesssys/constellation/pull/1469/files#diff-cc17dc16068258edef10efdad177053aa9c3298a6ba58c79f4d72c32b132b408) would not include the required `upgrade_test` target, since `upgrade_test.go` requires the build tag `e2e`.

Reply:
That makes sense, thanks. :)